src/main/java/com/amplitude/Response.java

```diff
@@ -0,0 +1,60 @@
+package com.amplitude;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import javax.net.ssl.HttpsURLConnection;
+import java.io.*;
+import java.net.URL;
+import java.util.List;
+
+public class Response {
+    protected int code;
+    protected Status status;
+    protected String error;
+    protected JSONObject SuccessBody;
+    protected JSONObject InvalidRequestBody;
+    protected JSONObject RateLimitBody;
+    protected static Response res;
```

Comment:
I'm sorry, why is this declared here? This should only be initialized once, right? `populateResponse` should only touch this variable once, based on some JSON?

Reply:
You are right, it didn't been used outside. Will move into populateResponse.